packages/liquidator/src/liquidator.js

```diff
@@ -134,6 +134,18 @@ class Liquidator {
           if (x === undefined) return true;
           return parseFloat(x) >= 0 && parseFloat(x) <= 100;
         }
+      },
+      contractType: {
+        value: undefined,
+        isValid: x => {
+          return x == "ExpiringMultiParty" || x == "Perpetual";
+        }
+      },
+      contractVersion: {
+        value: undefined,
+        isValid: x => {
+          return x == "1.2.0" || "1.2.1" || "1.2.2" || x == "latest";
```

Comment:
I feel like these hardcoded version values that are shared between this file and `index/liquidator.js` should be refactored into a shared dictionary.

Reply:
agreed. also just realized this has an error in it. I have corrected it. I want to leave this for a bit later though.